File: JAVASCRIPT/if_else/thirdmax.js
const prompt=require('prompt-sync')()
let a=Number(prompt('enter the value of a '))
let b=Number(prompt('enter the value of b '))
let c=Number(prompt('enter the value of c '))
let d=Number(prompt('enter the value of d '))
let max=0,smax=0,tmax=0;
// if(a>b){
//     max=a;
//     smax=b;
// }else{
//     max=b;
//     smax=a;
// }if(max>c){
//     if(smax>c){
//         smax=smax;
//         tmax=c;
//     }
//     else{
//         tmax=smax;
//         smax=c;
//     }
// }else{
//     tmax=smax;
//     smax=max;
//     max=c

// }
// if(max>d){
//     if(smax>d){
//         if(tmax>d){
//             tmax=tmax;
//         }else{
//             tmax=d;
//         }
//     }else{
//         tmax=smax;
//         smax=d;
//     }
// }else{
//     tmax=smax;
//     smax=max;
//     max=d;
// }
// console.log(tmax);

if(a>b){
    max=a;
    smax=b;
}else{
    max=b;
    smax=a;
}
if(max>c){
    if(smax>c){
        smax=smax;
        tmax=c;
    }else{
        tmax=smax;
        smax=c;
    }
}else{
    tmax=smax;
    smax=max;
    max=c;
}
if(max>d){
    if(smax>d){
        if(tmax>d){
            tmax=tmax;
        }else{
            tmax=d;
        }
    }else{
        tmax=smax;
        smax=d;
    }
}else{
    tmax=smax;
    smax=max;
    max=d;
}

console.log(tmax)

File: if_else/thirdmax.c
#include<stdio.h>
int main()
{
    int  a,b,c;
    printf("enter  the number a :");
    scanf("%d",&a);
    printf("enter the number b :");
    scanf("%d",&b);
    printf("enter the number c :");
    scanf("%d",&c);
    int tmax;
    if(a>=b && a>=c)
    {
        if(b>=c)
        {
            tmax=c;
            printf("third max is : %d \n",tmax);
        }else
        {
            tmax=b;
            printf("third max is :%d \n",tmax);
        }
    }else if(b>=c && b>=a)
    {
        if(c>=a)
        {
            tmax=a;
            printf("third max is :%d \n",tmax);
        }else
        {
            tmax=c;
            printf("third max is c :%d\n",tmax);
        }
    }else if(c>=a && c>=b)
    {
        if(a>=b)
        {
            tmax=b;
            printf("third max is :%d\n",tmax);
        }else
        {
            tmax=a;
            printf("third max is :%d\n",tmax);
        }
    }
    return 0;
}